app/services/problem-service.js

```diff
@@ -38,8 +38,10 @@ define(['./module'], function(services) {
                     data: null
                 };
                 return $http(req);
+            },
+            getProblemPhotos: function(problemID) {
+                return $http.get('http://127.0.0.1:8000/api/problems/' + problemID + '/photos');
```

Comment:
See show-problem.js. For static folder I added
$scope.photos_url = 'http://127.0.0.1:8000/static/photos/';.
getProblemPhotos - This service only fetch the data of uploaded photo.